CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,32 @@ internal class Testing
          */
 
         //Method
+
+        public void RunTest()
+        {
+            Game testGame = new Game();
+            Die testDie = new Die();
+            
+            // Die Testing
+            int testCases = 50;
+
+            for (int i = 0; i < testCases; i++)
+            {
+                int newRoll = testDie.Roll();
+                
+                Debug.Assert(newRoll < 7 && newRoll > 0, "Dice Roll detected as out of bounds.");
```

Comment:
While this testing does work there is potential to extract more information from errors which may occur e.g. Dice roll value lower than expected or higher than expected. This way if an error did occur finding its cause may be easier.